src/geovista/geoplotter.py

```diff
@@ -701,7 +705,7 @@ def _check(option: str) -> bool:
             # POI cartesian xyz
             self._poi = mesh.center
 
-        return super().add_mesh(mesh, **kwargs)
+        return super().add_mesh(mesh, **kwargs)  # type: ignore[misc]
```

Comment:
I managed to give `GeoPlotterBase` various methods using `Callable` type hints - since we know the intended use in multiple inheritance with a PyVista plotter - but I couldn't see an elegant way of doing anything analogous with references to `super()`, hence the `ignore`.